### ISSUE TITLE
Unable to hire employees/gang members

### ISSUE THREAD

**Issue by mockdot** [Qbox-project/qbx_management#7]
### Summary

Unable to hire employees/gang members

### Reproduction

1. Try to hire an employee or gang member on latest version.

The callback on line 169 of client/cl_boss.lua:     local players = lib.callback.await('qb-bossmenu:server:getplayers', false)

Is not returning the values properly, we have the following call:

```
-- Get closest player sv
lib.callback.register('qb-bossmenu:server:getplayers', function(source)
	local src = source
	return GetPlayers(src)
end)
```

```
---@param src number
---@return table
function GetPlayers(src)
    local players = {}
    local playerPed = GetPlayerPed(src)
    print('playerPed: ' .. playerPed)
    local pCoords = GetEntityCoords(playerPed)
    print('pCoords: ' .. pCoords)
    for _, v in pairs(QBCore.Functions.GetPlayers()) do
        local targetped = GetPlayerPed(v)
        print('targetped: ' .. targetped)
        local tCoords = GetEntityCoords(targetped)
        print('tCoords: ' .. tCoords)
        local dist = #(pCoords - tCoords)
        print('dist: ' .. dist)
        if playerPed ~= targetped and dist < 10 then
            print('v: ' .. v)
            local ped = QBCore.Functions.GetPlayer(v)
            print(ped)
            players[#players + 1] = {
                id = v,
                coords = GetEntityCoords(targetped),
                name = ped.PlayerData.charinfo.firstname .. " " .. ped.PlayerData.charinfo.lastname,
                citizenid = ped.PlayerData.citizenid,
                sources = GetPlayerPed(ped.PlayerData.source),
                sourceplayer = ped.PlayerData.source
            }
        end
    end

    table.sort(players, function(a, b)
        return a.name < b.name
    end)

    return players
end
```

[script:qbx-managemen] playerPed: 131329
[script:qbx-managemen] pCoords: vec3(462.039551, -985.608765, 30.712036)
[script:qbx-managemen] targetped: 135683
[script:qbx-managemen] tCoords: vec3(461.103302, -986.017578, 30.678345)
[script:qbx-managemen] dist: 1.022166967392
**[script:qbx-managemen] v: 3**
[script:qbx-managemen] SCRIPT ERROR: @qbx-management/server/main.lua:212: attempt to index a nil value (local 'ped')
[script:qbx-managemen] 

We can see the server ID of the player around is correct, but when running  local ped = QBCore.Functions.GetPlayer(v) it seems to return null. Been trying all day with this one, no luck.

### Expected behavior

Be able to hire

### Actual behavior

SCRIPT ERROR: @qbx-management/server/main.lua:212: attempt to index a nil value (local 'ped')

### Last Updated

today

### Custom Resources

no

### Resource Rename

qbx-management

**Comment by mockdot** [comment]
fixed #8

**Comment by Manason** [comment]
Issues should only be closed as completed when the PR that fixes it is merged to main. If you put into the PR description "fixes #8" it will do this automatically.

**Comment by mockdot** [comment]
Issue fixed with the above PR, closing this.
